media/css/firefox/browsers-products.scss

```diff
@@ -526,12 +528,22 @@ $url-download-link-hover: svg-url($download-link-hover);
 
 // box around form
 .mzp-c-emphasis-box.t-account {
-    @include light-links;
     background: $color-ink-80 linear-gradient($color-violet-50 -100%, $color-ink-80);
     color: $color-white;
     margin: 0 auto $v-grid-xs auto;
     position: relative;
 
+    a:link,
```

Comment:
The `light-links` mixin changed so inverted links now have color, but here and in a few other spots they need to be explicitly white. I've filed https://github.com/mozilla/protocol/issues/648